### PR TITLE
Update from v0.11.0 to v0.11.1

### PR DESCRIPTION
**Pull request by marcoesters**
Update Anaconda Project from version 0.11.0 to version 0.11.1

Addresses Jira ticket [DSNC-5208](https://anaconda.atlassian.net/browse/DSNC-5208).

# Package Info

Home: https://github.com/Anaconda-Platform/anaconda-project/
Source: https://github.com/Anaconda-Platform/anaconda-project/archive/v0.11.1.tar.gz
Documentation: https://anaconda-project.readthedocs.io/en/latest/
Change log: https://github.com/Anaconda-Platform/anaconda-project/compare/v0.11.0...v0.11.1

# Actions

* Update to version v0.11.1
* Update documentation

### CODE REVIEW

**Review by cbouss — APPROVED** [review]
LGTM

File: recipe/meta.yaml [hunk 1]
```diff
@@ -50,7 +50,7 @@ about:
     By adding an anaconda-project.yml to a project directory, a single 
     anaconda-project runcommand will be able to set up all dependencies 
     and then launch the project.
-  doc_url: http://anaconda-project.readthedocs.io
+  doc_url: https://anaconda-project.readthedocs.io/en/latest/
```
Comment (mcg1969) [inline recipe/meta.yaml]: I agree with the use of `https` here but I do not think you should follow the redirect. That's an implementation detail of readthedocs. The home page is the better choice IMO

Comment (marcoesters) [reply recipe/meta.yaml]: I got the opposite comment with my last PR in another feedstock. I will raise the issue in the slack channel, so we can come to an agreement.

Comment (marcoesters) [reply recipe/meta.yaml]: Redirect is removed.

**Pull request merged**